sklearn/model_selection/tests/test_search.py

```diff
@@ -332,6 +333,32 @@ def test_grid_search_groups():
         gs.fit(X, y)
 
 
+def test_future_warnings():
+    # Test that warnings are raised. Will be removed in 0.22
+
+    X = np.arange(100).reshape(10, 10)
+    y = np.array([0] * 5 + [1] * 5)
+    array = np.ones(10)
+    grid = {'C': array}
+
+    estimators = [GridSearchCV(LinearSVC(random_state=0), grid),
+                  RandomizedSearchCV(LinearSVC(random_state=0), grid)]
+    msg = ("Computing training scores may affect performance "
+           "significantly. This is the reason return_train_score will "
+           "change its default value from True (current behaviour) to "
+           "False in 0.22. Please set explicitly return_train_score to "
+           "get rid of this warning.")
+    for estimator in estimators:
+        search_with_warn = estimator.set_params(return_train_score="warn")
+        assert_warns_message(FutureWarning, msg, search_with_warn.fit, X, y)
+        with ignore_warnings():
```

Comment:
This should be unnecessary

Reply:
Removed the unnecessary ignore_warnings.